add created_at to visualization properties

gfiorav opened the pull request:
Fixes #9191 
Fixes https://github.com/CartoDB/cartodb/issues/9271

gfiorav commented:
Please CR, @javitonino

javitonino commented on lib/carto/tracking/events.rb:
```diff
@@ -3,11 +3,15 @@
+          privacy: visualization.privacy,
+          type: visualization.type,
+          created_at: created_at,
+          lifetime: Time.now.utc - created_at
```
This returns seconds, but days were requested in the ticket. Don't know if it is important (maybe seconds are better, more granularity)

gfiorav replied:
True! I asked in the original ticket in case we'd want' to reevaluate granularity. Sublime catch!

javitonino commented on lib/carto/tracking/events.rb:
```diff
@@ -3,11 +3,15 @@
+          vis_id: visualization.id,
+          privacy: visualization.privacy,
+          type: visualization.type,
+          created_at: created_at,
```
Check that this date format is parseable by Segment. I guess it is, but I don't know how this works exactly.

gfiorav replied:
yup, we've been using it already! https://github.com/CartoDB/cartodb/blob/master/lib/carto/tracking/events.rb#L56

javitonino replied:
Awesome :sparkles:

javitonino commented:
Take into account the comments you want, and ship it!

gfiorav commented:
I changed the units to days (with decimal) and added some properties to the event general properties. Review again please @javitonino and let's put this in prod as soon as posible!

javitonino commented on lib/carto/tracking/events.rb:
```diff
@@ -3,11 +3,16 @@
-      def visualization_properties(table_visualization, origin: nil)
+      def visualization_properties(visualization, origin: nil)
+        created_at = visualization.created_at
+        lifetime_in_days_with_decimal = (Time.now.utc - created_at).to_f / 60 / 60 / 24
```
Ruby not having proper timediff objects is a bit disappointing. :cry: 

Anyway, can you extract this to a function? It is repeated in `event_properties`.

javitonino commented:
Other than the proposed refactor (extract timediff function), nothing else to add. Ship when :green_apple:

javitonino commented:
:shipit:
